Misc/NEWS.d/next/Library/2025-02-02-00-30-09.gh-issue-85162.BNF_aJ.rst

```diff
@@ -0,0 +1,6 @@
+The :mod:`http.server` module now includes built-in support for HTTPS
```

Comment:
This requires more than just a blurb. it's a full fledged new feature so a What's New entry is needed.